addons/mail/models/mail_followers.py

```diff
@@ -142,29 +146,25 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None, cids
         LEFT JOIN res_users users ON users.partner_id = partner.id AND users.active
         LEFT JOIN res_groups_users_rel groups_rel ON groups_rel.uid = users.id
         LEFT JOIN res_groups groups ON groups.id = groups_rel.gid
-        WHERE EXISTS (
-            SELECT partner_id FROM sub_followers
-            WHERE sub_followers.channel_id IS NULL
-                AND sub_followers.partner_id = partner.id
-                AND (coalesce(sub_followers.internal, false) <> TRUE OR coalesce(partner.partner_share, false) <> TRUE)
-        ) %s
+        JOIN sub_followers ON sub_followers.partner_id = partner.id
```

Comment:
I know it makes no difference but not having a pair of braces around the multi-term ON condition makes me cringe ;)